### ISSUE TITLE
AttributeError: 'set' object has no attribute 'country_code'

### ISSUE THREAD

**Issue by arogl** [snejus/beets-bandcamp#10]
```
bandcamp: Failed obtaining https://skeptikmetal.bandcamp.com/album/signature-in-blood-feat-maddie-veil-single
Traceback (most recent call last):
  File "/home/arogl/.local/lib/python3.8/site-packages/beetsplug/bandcamp/__init__.py", line 271, in handle
    return call()
  File "/home/arogl/.local/lib/python3.8/site-packages/beetsplug/bandcamp/_metaguru.py", line 414, in album
    return self.albuminfo(include_all)
  File "/home/arogl/.local/lib/python3.8/site-packages/beetsplug/bandcamp/_metaguru.py", line 372, in albuminfo
    "country": self.country,
  File "/home/arogl/.local/lib/python3.8/site-packages/beetsplug/bandcamp/_metaguru.py", line 213, in country
    or subdivisions.lookup(ascii_name).country_code
AttributeError: 'set' object has no attribute 'country_code'
```

Might need to check objects/mappings exist before trying to assign them?

Beets v1.5.0 (commit 04ea754d00e2873ae9aa2d9e07c5cefd790eaee2)
Beetcamp V0.6.0

**Comment by snejus** [comment]
Sorry, completely missed out on this issue! Will have a look in a minute.

**Comment by snejus** [comment]
Sorry for taking ages - this has been fixed in `0.7.0`.
